distributed/client.py

```diff
@@ -3455,6 +3455,44 @@ def restart(self, timeout=no_default, wait_for_workers=True):
             self._restart, timeout=timeout, wait_for_workers=wait_for_workers
         )
 
+    def restart_workers(self, workers: list[str]):
```

Comment:
`Nanny.restart` is taking a timeout argument. We should allow this to be provided here